polkadot/parachain/test-parachains/Cargo.toml

```diff
@@ -1,21 +0,0 @@
-[package]
-name = "test-parachains"
```

Comment:
I am not sure about purpose of this module, but if it is not used, I am ok with removing it